dynamic_rest/fields/fields.py

```diff
@@ -135,7 +135,7 @@ def bind(self, *args, **kwargs):
                 )
         ):
             self.required = False
-        if 'allow_null' not in self.kwargs and getattr(
+        if 'allow_null' in self.kwargs and getattr(
```

Comment:
Definitely some funky business going on here. If I re-insert `not in`, then I have some tests in dependents of dynamic-rest failing because model fields marked "null" aren't treated as such.